metricbeat/module/docker/diskio/data.go

```diff
@@ -0,0 +1,30 @@
+package diskio
+
+import (
+	"github.com/elastic/beats/libbeat/common"
+)
+
+func eventsMapping(blkioStatsList []BlkioStats) []common.MapStr {
+	myEvents := []common.MapStr{}
+	for _, blkioStats := range blkioStatsList {
+		myEvents = append(myEvents, eventMapping(&blkioStats))
+	}
+	return myEvents
+}
+func eventMapping(myBlkioStats *BlkioStats) common.MapStr {
+	event := common.MapStr{
+		"@timestamp": myBlkioStats.Time,
+		"type":       "blkio",
+		"container": common.MapStr{
+			"id":     myBlkioStats.MyContainer.Id,
+			"name":   myBlkioStats.MyContainer.Name,
+			"labels": myBlkioStats.MyContainer.Labels,
+		},
+		"blkio": common.MapStr{
+			"readsPS":  myBlkioStats.reads,
```

Comment:
For the event naming, check our fields conventions: https://www.elastic.co/guide/en/beats/libbeat/5.0/event-conventions.html

I assume PS stands for per second? In the above case I would probably go with jus `blkio.reads`, `blkio.writes` etc. as all are in the same unit.